parts/k8s/kubernetesagentvars.t

```diff
@@ -24,4 +24,5 @@
     "{{.Name}}osImageSKU": "[parameters('{{.Name}}osImageSKU')]",
     "{{.Name}}osImagePublisher": "[parameters('{{.Name}}osImagePublisher')]",
     "{{.Name}}osImageVersion": "[parameters('{{.Name}}osImageVersion')]",
-
+    "{{.Name}}osImageName": "[parameters('{{.Name}}osImageName')]",
```

Comment:
Opened https://github.com/Azure/acs-engine/pull/2613 for pushing these changes; we should already have them in this repo (mea culpa).

Reply:
#2613 has been merged :)

Reply:
fixed